src/pages/Contact.js

```diff
@@ -1,7 +1,176 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { TextField, Button, Stack, Box, Modal, Paper } from "@mui/material";
+import { useForm, Controller } from "react-hook-form";
+import { DevTool } from "@hookform/devtools";
+import axios from "axios";
 
 function Contact() {
-  return <div>Contact</div>;
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      firstName: "",
+      lastName: "",
+      email: "",
+      orderInfo: "",
+    },
+  });
+  const [dataFromServer, setDataFromServer] = useState("");
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  useEffect(() => {
+    const fetDataFromServer = async () => {
+      try {
+        const response = await axios.get("http://localhost:9006/api/data");
+
+        setDataFromServer(response.data);
+      } catch (error) {
+        console.error("Error fetching data from the server", error);
+      }
+    };
+    fetDataFromServer();
+  }, []);
+
+  const handleModalClose = () => {
+    setIsModalOpen(false);
+  };
+
+  const onSubmit = async (data) => {
+    const endpoint = "http://localhost:9006/send-email";
+    try {
+      const response = await axios.post(endpoint, data, {
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+      if (response.status === 200) {
+        console.log("Email sent!");
```

Comment:
Let's remove console logs as it is working as intended in Front end side of things. The Server Console Logs can stay

Reply:
got it!